packages/sdk-core/src/__test__/proving.spec.ts

```diff
@@ -0,0 +1,287 @@
+// Copyright 2022 @webb-tools/
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable camelcase */
+
+// eslint-disable-next-line camelcase
+import { JsNoteBuilder, MTBn254X5, OutputUtxoConfig, setupKeys, verify_js_proof } from '@webb-tools/wasm-utils/njs/wasm-utils-njs.js';
+import { expect } from 'chai';
+
+import { hexToU8a, u8aToHex } from '@polkadot/util';
+
+import { ProvingManagerSetupInput, ProvingManagerWrapper } from '../proving/index.js';
+
+function generateVAnchorNote (
+  amount: number,
+  chainId: number,
+  outputChainId: number,
+  index?: number
+) {
+  const noteBuilder = new JsNoteBuilder();
+
+  noteBuilder.protocol('vanchor');
+  noteBuilder.version('v2');
+  noteBuilder.backend('Arkworks');
+  noteBuilder.hashFunction('Poseidon');
+  noteBuilder.curve('Bn254');
+
+  noteBuilder.sourceChainId(String(chainId));
+  noteBuilder.targetChainId(String(outputChainId));
+  noteBuilder.width(String(5));
+  noteBuilder.exponentiation(String(5));
+  noteBuilder.denomination(String(18));
+  noteBuilder.amount(String(amount));
+  noteBuilder.tokenSymbol('WEBB');
+  noteBuilder.targetIdentifyingData('');
+  noteBuilder.sourceIdentifyingData('');
+  const note = noteBuilder.build();
+
+  if (index !== undefined) {
+    note.mutateIndex(String(index));
+  }
+
+  return note;
+}
+
+const vanchorBn2542_2_2 = setupKeys('vanchor', 'Bn254', 2, 2, 2);
```

Comment:
Can we add another underscore here? `vanchorBn254_2_2_2` and `vanchorBn254_2_16_2`